config/odp-linux-generic.conf

```diff
@@ -37,6 +37,9 @@ shm: {
 	# Allocate internal shared memory using a single virtual address space.
 	# Set to 1 to enable using process mode.
 	single_va = 0
+
+ 	# Amount of memory pre-reserved for ODP_SHM_SINGLE_VA usage in bytes
+	single_va_size = 67108864
```

Comment:
It's better to specify this in units of megabytes rather than bytes both for ease of use and to avoid the false impression that you're offering byte-level granular control. So `single_va_size = 64` would read much cleaner.  

Note that you're specifying 64MB here even though the commit comment says you're lowering the limit to 50MB, which shows how trying to specifying things in bytes vs. megabytes leads to specification errors.

Reply:
KB seems too small and artificially granular. Are there really any cases when we'd specify less than 1MB increments of shared memory? It also doesn't help much when specifying memory in the GB range. We could support MB/GB suffixes but that seems overkill on the parsing side.

Reply:
In case of chip's internal memory, kilobyte level definition may be useful. Since this is a configuration option I'd prioritize configurability over readability.

Kilobytes are fine for linux:
E.g. /proc/meminfo
```
MemTotal:         492836 kB
MemFree:          304820 kB
MemAvailable:     414916 kB
```

Reply:
Linux's use of kilobytes reflects the legacy of 1990s-vintage small desktop systems as well as older embedded environments. But Linux itself needs several megabytes these days to boot, so it's a bit of an anachronism. 

In the case of ODP, are there any use cases where we'd want/need to specify sub-MB granularity? Especially for this intended use of shared large memory areas for supporting process mode, 1MB is actually a fairly small granule. The issue is what happens when due to typos or carelessness a `single_va_size = 67108872` is specified? Is that going to work wherever odp-linux finds itself running?